src/Avalonia.Controls/ScrollViewer.cs

```diff
@@ -448,6 +448,38 @@ public void LineRight()
             Offset += new Vector(_smallChange.Width, 0);
         }
 
+        /// <summary>
+        /// Scrolls the content upward by one page.
+        /// </summary>
+        public void PageUp()
```

Comment:
This code doesn't seem to support ILogicalScrollable. See ScrollContentPresenter.cs sources for more details.

Reply:
I think this should handle `ILogicalScrollable`. The logical scrolling stuff should be handled in `ScrollContentPresenter` and by the time the info gets to `ScrollViewer` (via two-way bindings) the offset/extent/viewport will have been calculated according to logical scrolling.

Reply:
@grokys Practically. Do you suggest replicating the methods in the ILogicalScrollable interface and using these in the ScrollViewer?

Reply:
Not 100% sure what you mean, but I don't think I was suggesting that, no ;)